skrub/_agg_joiner.py

```diff
@@ -49,87 +50,68 @@ def split_num_categ_operations(operations: list[str]) -> tuple[list[str], list[s
     return num_operations, categ_operations
 
 
-def check_missing_columns(
-    X,
-    columns,
-    error_msg,
-):
-    """All elements of main_key must belong to the columns of X.
-
-    Parameters
-    ----------
-    X : DataFrameLike
-        Input data.
-
-    main_key : list of string
-        Key used to perform join on X.
-    """
-    missing_cols = set(columns) - set(X.columns)
-    if len(missing_cols) > 0:
-        raise ValueError(error_msg)
-    return
-
-
 class AggJoiner(BaseEstimator, TransformerMixin):
-    """Aggregate auxiliary dataframes before joining them on a base dataframe.
```

Comment:
while you're doing some cleanup could you change `BaseEstimator, TransformerMixin` to `TransformerMixin, BaseEstimator` here and in aggtarget?